docs/DeveloperGuide.md

```diff
@@ -857,10 +856,10 @@ testers are expected to do more *exploratory* testing. Each test case is to be e
 
 1. Adding Login screen
 
-   1. Prerequisites: You are in deliverable mode. Login screen not already added. If added, delete it.
+   1. Prerequisites: You are in deliverable mode. The deliverable `Login screen` is not already added. If added, delete it.
 
-   1. Test case: `add t/Login screen m/1.0 by/12-12-2020 23:59 d/Must include username and password fields c/John Martin, Abby Li`<br>
-      Expected: Login screen to appear in the list of deliverables and expanded in right panel.
+   1. Test case: `add t/Login screen by/10-10-2020 18:00 m/1.1 c/Jordan Woods, Betsy Crowe d/Include email and password fields`<br>
+      Expected: The deliverable `Login screen` is added to the list of deliverables and is expanded in the right panel.
```

Comment:
I think we should settle on the terms. (In UG, might call it as view panel in the intro section)